plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceDescriptor.java

```diff
@@ -1087,6 +1087,10 @@ private boolean askForSSHJumpServerPassword(@NotNull DBWHandlerConfiguration tun
     }
 
     public void openDataSource(DBRProgressMonitor monitor, boolean initialize) throws DBException {
+        if (this.getDriver().isSingleConnection()) {
```

Comment:
Maybe its better to set default value of `forceUseSingleConnection` in constructor?